code/api/views.py

```diff
@@ -13,10 +16,12 @@
 import logging
 
 from cmput404.constants import HOST, API_PREFIX
+from socialDistribution.forms import PostForm
 from socialDistribution.models import *
 from .decorators import authenticate_request, validate_node
 from .parsers import url_parser
-from .utility import getPaginated
+from .utility import getPaginated, makePost
+from socialDistribution.dispatchers import dispatch_post
```

Comment:
Do we need this import? Just asking because it's for socialDistribution but we're in the api app.